content/soda-gettingstarted/soda-lab-setup/k8s-setup.md

```diff
@@ -0,0 +1,230 @@
+---
+title: Kubernetes Setup
+description: "Setup Kubernetes testing lab for SODA enviroment"
+weight: 70
+---
+
+## Kubernetes
+
+Kubernetes (K8s) is an open-source system for automating deployment,
+scaling, and management of containerized applications.
+
+Kubernetes is a portable, extensible, open-source platform for managing
+containerized workloads and services, that facilitates both declarative
+configuration and automation. It has a large, rapidly growing ecosystem.
+Kubernetes services, support, and tools are widely available.
+
+### Features
+
+-   Service discovery and load balancing
+-   Storage orchestration
+-   Automated rollouts and rollbacks
+-   Automatic bin packing
+-   Self-healing
+
+### Local Lab View
+
+![image][]
+
+source: [Rancher Documentation][]
+
+## Getting Started
+
+![image][0]
+
+### Section:1
+
+
+Final vagrant and script files for the various nodes to get quickly started (Ready to go files)
+
+| Sno | Hostname         |
+|-----|------------------|
+| 1   | master.osive.lab |
+| 2   | node1.osive.lab  |
+| 3   | node2.osive.lab  |
+
+[Download files here](../soda-lab-setup/k8s-scripts.zip)
```

Comment:
This path points to 404. The zip file is in the same directory as the document. Maybe change to `./k8s-scripts.zip`.
Same for other zip files.
Consider placing them in the static folder in the documentation root.